projects/clr/rocclr/device/rocm/rocmemory.hpp

```diff
@@ -126,7 +126,9 @@ class Memory : public device::Memory {
 
   // Free / deregister device memory.
   virtual void destroy() = 0;
-  hsa_status_t interopMapBuffer(amd::Os::FileDesc fdn);
+
+  // Map interop buffer
+  hsa_status_t interopMapBuffer(hsa_handle_t fdn, hsa_handle_type_t handleType);
```

Comment:
the interface doesn't have default value for the type. It will fail Linux compilation.